.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Feature Request
+about: Use this template to request a feature or enhancement.
+---
+
+- What would you like implemented? What do you want that Twilight lacks?
```

Comment:
Might want to add something to ask if it is a discord api feature that is unimplemented, as we should aim for 100% api coverage and no matter what it'll likely get added then